### PR TITLE
Revise TSO related Grafana monitoring metrics description

### PR DESCRIPTION
**Pull request by lilin90**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/18463
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-monitoring.md [hunk 1]
```diff
@@ -69,11 +69,11 @@ Number of queries using plan cache per second in all TiDB instances
 ### KV/TSO Request OPS
 
 - kv request total: Total number of KV requests per second in all TiDB instances
-- kv request by type: Number of KV requests per second in all TiDB instances based on such types as `Get`, `Prewrite`, and `Commit`.
-- tso - cmd: Number of `tso cmd` requests per second in all TiDB instances
-- tso - request: Number of `tso request` requests per second in all TiDB instances
+- kv request by type: Number of KV requests per second in all TiDB instances based on such types as `Get`, `Prewrite`, and `Commit`
+- tso - cmd: Number of gRPC requests per second in all TiDB instances; each gRPC request contains a batch of TSO requests
```
Comment (lilin90) [inline dashboard/dashboard-monitoring.md:73]:
```suggestion
- tso - cmd: Number of gRPC requests per second that TiDB sends to PD in all TiDB instances; each gRPC request contains a batch of TSO requests
```

File: grafana-performance-overview-dashboard.md [hunk 2]
```diff
@@ -67,11 +67,11 @@ Number of commands processed by all TiDB instances per second based on type
 ### KV/TSO Request OPS
 
 - kv request total: Total number of KV requests per second in all TiDB instances
-- kv request by type: Number of KV requests per second in all TiDB instances based on such types as `Get`, `Prewrite`, and `Commit`.
-- tso - cmd: Number of `tso cmd` requests per second in all TiDB instances
-- tso - request: Number of `tso request` requests per second in all TiDB instances
+- kv request by type: Number of KV requests per second in all TiDB instances based on such types as `Get`, `Prewrite`, and `Commit`
+- tso - cmd: Number of gRPC requests per second in all TiDB instances; each gRPC request contains a batch of TSO requests
```
Comment (lilin90) [inline grafana-performance-overview-dashboard.md:71]:
```suggestion
- tso - cmd: Number of gRPC requests per second that TiDB sends to PD in all TiDB instances; each gRPC request contains a batch of TSO requests
```

**Review by JmPotato — APPROVED** [review]

File: grafana-tidb-dashboard.md [hunk 3]
```diff
@@ -133,10 +133,10 @@ The following metrics relate to requests sent to TiKV. Retry requests are counte
 - PD Client CMD OPS: the statistics of commands executed by PD Client per second
 - PD Client CMD Duration: the time it takes for PD Client to execute commands
 - PD Client CMD Fail OPS: the statistics of failed commands executed by PD Client per second
-- PD TSO OPS: the number of TSO that TiDB obtains from PD per second
+- PD TSO OPS: the number of gRPC requests per second that TiDB sends to PD (cmd) and the number of TSO requests (request); each gRPC request contains a batch of TSO requests
 - PD TSO Wait Duration: the time that TiDB waits for PD to return TSO
-- PD TSO RPC duration: the duration from the time that TiDB sends request to PD (to get TSO) to the time that TiDB receives TSO
-- Start TSO Wait Duration: the duration from the time that TiDB sends request to PD (to get `start TSO`) to the time that TiDB receives `start TSO`
+- PD TSO RPC duration: the duration from the time that TiDB sends gRPC requests to PD to get TSO to the time that TiDB receives the gRPC response from PD
+- Async TSO Duration: the duration from the time that TiDB wants to get TSO to the time that TiDB actually starts to wait for PD to return TSO
```
Comment (qiancai) [inline grafana-tidb-dashboard.md:139]:
```suggestion
- Async TSO Duration: the duration from the time that TiDB prepares to get TSO to the time that TiDB actually starts to wait for PD to return TSO
```

File: grafana-performance-overview-dashboard.md [hunk 4]
```diff
@@ -67,11 +67,11 @@ Number of commands processed by all TiDB instances per second based on type
...
 - kv request total: Total number of KV requests per second in all TiDB instances
-- kv request by type: Number of KV requests per second in all TiDB instances based on such types as `Get`, `Prewrite`, and `Commit`.
-- tso - cmd: Number of `tso cmd` requests per second in all TiDB instances
-- tso - request: Number of `tso request` requests per second in all TiDB instances
+- kv request by type: Number of KV requests per second in all TiDB instances based on such types as `Get`, `Prewrite`, and `Commit`
+- tso - cmd: Number of gRPC requests per second that TiDB sends to PD in all TiDB instances; each gRPC request contains a batch of TSO requests
+- tso - request: Number of TSO requests per second in all TiDB instances
 
-Generally, dividing `tso - cmd` by `tso - request` yields the average batch size of requests per second.
+Generally, `tso - request` divided by `tso - cmd` is the average size of the TSO request batch per second.
```
Comment (qiancai) [inline grafana-performance-overview-dashboard.md:74]:
```suggestion
Generally, `tso - request` divided by `tso - cmd` is the average size of TSO request batches per second.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: dashboard/dashboard-monitoring.md [hunk 5]
```diff
@@ -69,11 +69,11 @@ Number of queries using plan cache per second in all TiDB instances
...
 - kv request total: Total number of KV requests per second in all TiDB instances
-- kv request by type: Number of KV requests per second in all TiDB instances based on such types as `Get`, `Prewrite`, and `Commit`.
-- tso - cmd: Number of `tso cmd` requests per second in all TiDB instances
-- tso - request: Number of `tso request` requests per second in all TiDB instances
+- kv request by type: Number of KV requests per second in all TiDB instances based on such types as `Get`, `Prewrite`, and `Commit`
+- tso - cmd: Number of gRPC requests per second that TiDB sends to PD in all TiDB instances; each gRPC request contains a batch of TSO requests
+- tso - request: Number of TSO requests per second in all TiDB instances
 
-Generally, dividing `tso - cmd` by `tso - request` yields the average batch size of requests per second.
+Generally, `tso - request` divided by `tso - cmd` is the average size of the TSO request batch per second.
```
Comment (qiancai) [inline dashboard/dashboard-monitoring.md:76]:
```suggestion
Generally, `tso - request` divided by `tso - cmd` is the average size of TSO request batches per second.
```

**Pull request merged**